selenium_tests/course_info/course_info_add_people_tests.py

```diff
@@ -51,3 +52,94 @@ def test_add_person(self, test_case_id, test_user, canvas_role, role_id):
         self.api.remove_user(self.test_settings['test_course']['cid'],
                              test_user, role_id)
 
+
+class MultipleAddPeopleTests(CourseInfoBaseTestCase):
+
+    def _load_find_info_tool(self):
+        """
+        Common code: This method loads up the find_info_tool and goes to the
+        People's page
+        """
+        self._load_test_course()
+        self.detail_page.go_to_people_page()
+        self.assertTrue(self.people_page.is_loaded())
+
+
+    def test_multi_user_add_unsuccessful(self):
+        """
+        TLT-2574
```

Comment:
Updated.  Generally, I'll stick to referring to AC as it is more stable.  Will refer to specific TC when applicable (talk to Robert - he will be expanding on a few), but keep in mind that TC numbering is generally more fluid than AC (so the numbering could be changed when TC's are added/removed/re-ordered).   Fixed for these tests - thanks!